### ISSUE TITLE
Uncaught Error: Git exited with unexpected code: 1

### ISSUE THREAD

**Issue by Gabb1995** [Stepsize/atom-better-git-blame#5]
Steps:
Opening atom

**Atom**: 1.20.1 x64
**Electron**: 1.6.9
**OS**: Ubuntu 17.04
**Thrown From**: [better-git-blame](https://github.com/Stepsize/atom-better-git-blame) package 0.1.3

### Stack Trace

Uncaught Error: Git exited with unexpected code: 1

```
At /home/gabriel/.atom/packages/better-git-blame/dist/bundle.js:24345

Error: Git exited with unexpected code: 1
    at ChildProcess.child.on.exitCode (/packages/better-git-blame/dist/bundle.js:24345:19)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
```

### Commands

```
  2x -0:12.6.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-pro-vivid-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
aligner 1.2.4 
aligner-css 1.2.1 
aligner-javascript 1.3.0 
aligner-php 1.2.0 
apex-ui 1.0.1 
atom-autocomplete-php 0.25.6 
atom-limelight 0.1.2 
atom-symfony2 0.3.0 
atom-sync 0.7.3 
auto-close-html2 0.6.1 
better-git-blame 0.1.3 
busy-signal 1.4.3 
chubrub 0.2.0 
docblockr 0.11.0 
file-icons 2.1.12 
folded-code-preview 0.2.4 
git-plus 7.9.3 
hyperclick 0.1.5 
intentions 1.1.5 
linter 2.2.0 
linter-phpcs 1.6.7 
linter-ui-default 1.6.10 
minimap 4.29.7 
minimap-autohide 0.10.1 
minimap-git-diff 4.3.1 
minimap-linter 2.1.1 
minimap-selection 4.5.0 
naturepurple-dark-syntax 0.1.1 
ninja-atom-syntax 0.1.2 
one-dark-pro-vivid-syntax 2.3.0 
one-dark-vivid-syntax 1.8.0 
pdf-view 0.59.0 
php-hyperclick 1.2.1 
php-twig 4.0.0 
pigments 0.40.2 
pochinki-syntax 0.1.0 
sync-settings 0.8.3 
t-rex-syntax 0.1.0 
```

**Comment by nomeyer** [comment]
Hey @Gabb1995, thanks for letting us know 😀  We're working on a fix, expect an update within the next couple hours.

**Comment by pthm** [comment]
This has been resolved with #6 and released as version 0.1.5

Tested on: macOS 10.12.6 & Ubuntu 17.04
